### ISSUE TITLE
Create scheduling tool for customer pick-ups

### ISSUE THREAD

**Issue by aubrel** [meitar/flora-posidonia#7]
Create customer-facing calendar that can be used to schedule customer payments and pick-ups after their orders are completed.

**Comment by meitar** [comment]
I don't think this needs to be created (yet). Have you tried playing around with one or more of the following plugins on the test site?

* [Woocommerce order delivery or pickup with date and time](https://wordpress.org/plugins/byconsole-woo-order-delivery-time/)
* [WooCommerce Local Pickup Time Select](https://wordpress.org/plugins/woocommerce-local-pickup-time-select/)

These might not do everything you need yet but you should try them out to see if they do.

Also, this doesn't need to be done "by launch" since, IIRC, you will be using email to communicate with a customer about when they'd like to pick up their order, and that simply needs a modification to the default "Customer invoice" email, discussed in #4. So, I'm going to move this out of the Launch milestone.
